.changeset/cuddly-mugs-invite.md

```diff
@@ -0,0 +1,6 @@
+---
+'@lit-labs/analyzer': minor
```

Comment:
should these be patches (== minor in pre-release range)?

Reply:
probably right, i'll update